tasks_downstream.py

```diff
@@ -660,16 +661,18 @@ def test(
         raise exceptions.ParseError(
             msg="Available modes are 'init' or 'update'. See --help for details."
         )
-    odoo_command.append(modules)
     # Skip test in some modules
     modules_list = modules.split(",")
-    for m_to_skip in skip or []:
+    for m_to_skip in skip.split(","):
+        if not m_to_skip:
+            continue
```

Comment:
Passing in `--skip ,,,` should be supported? 🤔

Reply:
Well, this was covering the case where `skip` is `""` (default) -> the list will be `[""]` -> the first `m_to_skip` will be `""`, which shouldn't be parsed. Is it worth preventing that specific situation and throw an error? `--skip ,,,` won't cause damage, just won't be parsed...